src/ol_metadata.c

```diff
@@ -138,6 +138,54 @@ ol_metadata_new_from_variant (GVariant *variant)
     }
   }
   g_variant_iter_free (iter);
+
+  // add lakedai 2020/10/02，parse title to help search
```

Comment:
1 - remove your name from this comment, git will remember who did this changes, and you'll be listed as a contributor in the repository;
2 - add to the comment, what are the conditions for this to trigger.